policy/test/inputs/parsec-cca-evidence.json

```diff
@@ -0,0 +1,52 @@
+            "cca-platform-challenge": "tZc8touqn8VVWHhrfsZ/aeQN9bpaqSHNDCf0BYegEeo=",
+            "cca-platform-config": "AQcGBQQDAgEADw4NDAsKCQgXFhUUExIREB8eHRwbGhkY",
+            "cca-platform-hash-algo-id": "sha-256",
+            "cca-platform-implementation-id": "f0VMRgIBAQAAAAAAAAAAAAMAPgABAAAAUFgAAAAAAAA=",
+            "cca-platform-instance-id": "AQcGBQQDAgEADw4NDAsKCQgXFhUUExIREB8eHRwbGhkY",
+            "cca-platform-lifecycle": 12291,
+            "cca-platform-profile": "http://arm.com/CCA-SSD/1.0.0",
+            "cca-platform-service-indicator": "whatever.com",
+            "cca-platform-sw-components": [{
+                "measurement-description": "TF-M_SHA256MemPreXIP",
+                "measurement-type": "BL",
+                "measurement-value": "BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=",
+                "signer-id": "BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=",
+                "version": "3.4.2"
+            }, {
+                "measurement-type": "M1",
+                "measurement-value": "BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=",
+                "signer-id": "BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=",
+                "version": "1.2"
+            }, {
+                "measurement-type": "M2",
+                "measurement-value": "BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=",
+                "signer-id": "BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=",
+                "version": "1.2.3"
+            }, {
+                "measurement-type": "M3",
+                "measurement-value": "BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=",
+                "signer-id": "BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=",
+                "version": "1"
+            }]
+        },
+        "cca.realm": {
+            "cca-realm-challenge": "cqJXkxzhxf4rU34/UCEvka5SlX1AKjsp7FQO+k7L0aRgp4xorPOmOXDAEwM5y/5Y2ePbs84RCTRgx1L8iQqwIQ==",
+            "cca-realm-extensible-measurements": ["AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=", "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=", "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=", "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="],
+            "cca-realm-hash-algo-id": "sha-256",
+            "cca-realm-initial-measurement": "X5A2VVSw+obQdbbWpOpYZtsXk9S06ZO7UuVk1yefEXg=",
+            "cca-realm-personalization-value": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==",
+            "cca-realm-public-key": "BHb5iAkb5YXtQYAa7Pq4WFSMYwV+FrDmdhILvQ0vnCngVsXUGgEw65whUXiZ3CMUayjhsGK9PqSzFf0hnxy7Uoy250ykm+Fnc3NPYaHKYQMbK789kY8vlP/EIo5QkZVErg==",
+            "cca-realm-public-key-hash-algo-id": "sha-256"
+        },
+        "kat": {
+            "akpub": "{\"KeyType\":2,\"KeyID\":null,\"KeyOps\":null,\"BaseIV\":null,\"Algorithm\":0,\"Curve\":1,\"K\":null,\"X\":\"m//bpnUzLJu7XeKy2gn4GEdKmJrCACP4KMFiC6gPYdg=\",\"Y\":\"yDnCcjDfAZhA06RclhhvjQcZPrxDqEHg3eLEY/ZMShU=\",\"D\":null}",
+            "nonce": "AAECAwQFBgc="
```

Comment:
will this serialisation break when https://github.com/veraison/go-cose/pull/158 is merged?

Reply:
Since go-cose/COSE_Key makes no promise about its JSON serialisation, using it here is fragile.  go-cose/COSE_Key has a CBOR serialisation which is stable.  I suggest we use that and base64 encode the result.

Reply:
Thinking a bit more, regardless of whether the JSON serialization is provided by the library or not, even if the CBOR serialization is used, and if the Internal structure of the `COSE_Key` is changed between revisions, then the associated CBOR serialization will change. Yes, the code in both cases, does not change but the unit tests still needs to modify complying to the new serialization, in both the cases.

Reply:
> Thinking a bit more, [...] if the Internal structure of the `COSE_Key` is changed between revisions, then the associated CBOR serialization will change.

No, it won't.  Why should it?

Reply:
Let me clarify what I meant. Whether one serializes CBOR or JSON the input test vectors required for the test to run needs to change as soon as  the `COSE_Key`  structure changes with the new revision and one picks the new revision of go-cose.

Reply:
Can you explain why do you think the CBOR serialisation of the COSE_Key changes if the go-cose `Key` structure changes?

Reply:
If a new map key gest added to below: then the CBOR would change isn't it?

// Any additional parameter (label,value) pairs.
	Params map[interface{}]interface{}

Reply:
> If a new map key gets added to below: then the CBOR would change isn't it?

no, if it's not set it's not serialised.

Reply:
Completed the translation!